ui/src/pages/index.tsx

```diff
@@ -1,10 +1,11 @@
 import React from 'react'
 import { Link } from 'react-router-dom'
-import { Section, LogoStrip, Icon } from 'components'
+import { Section, LogoStrip, Icon, Button } from 'components'
 import { TerminalScreen, Screen } from 'figures'
 import ecommerceImg from 'images/ecommerce.png'
 import lutImg from 'images/lut-scott.png'
 import { SEO } from '../utilities/SEO'
+import { GraphQLEnumType } from 'graphql'
```

Comment:
Can be removed, this was am accidental import when we were pair coding before

Reply:
roger roger roger roger roger roger... you get it